src/nginxconfig/templates/domain_sections/logging.vue

```diff
@@ -55,6 +81,34 @@ THE SOFTWARE.
                                 {{ $t('templates.domainSections.logging.enableForThisDomain') }}
                             </PrettyCheck>
                         </div>
+                        <div v-if="$props.data.errorLog.computed" class="control field is-horizontal is-expanded">
+                            <input
+                                v-model="errorLogPath"
+                                class="input"
+                                type="text"
+                                :placeholder="$props.data.errorLogPath.computed"
+                            />
+                        </div>
```

Comment:
Ditto on all three thoughts here (is-changed indicator, watcher for the default computed, parent field alignment)